Update README.md to include sentence of how clients shall interpret last page (#450)
radiantearth/stac-api-spec: item-search/README.md

Implementations may also add link relations `prev`, `first`, and `last`, though these are not required and may
be infeasible to implement in some data stores.

> Clients shall interpre the absence of `prev` or `last` link relations to indicate the response represents respectively the first and last page of results.

Reviewer: As prev is not required, we can't interpret it as first page.

```suggestion
Clients shall interpret the absence of a link with the relation type `next` as an indication that the response represents the last page of results.
```
